client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -32,25 +36,50 @@ class EditorDrawerFeaturedImage extends Component {
 		stats.recordEvent( 'Featured image removed' );
 	}
 
+	onFilesDrop = ( files ) => {
+		Dispatcher.register( function( payload ) {
```

Comment:
I wouldn't have expected to have to extend the old dispatcher. Questions:
- why is it needed?
- if it is, can we extend a preëxisting `Dispatcher.register` callback?
- if it is, could this be better accomplished with one-off subscriptions (e.g. `eventEmitter.once( cb )`)?
- if it is, could this be better accomplished by relaying these into Redux?
- I'm assuming it's a temporary thing, but right now this obviously will add a new handler to `Dispatcher` on each `filesdrop` event.